website/docs/automation/controlplanes/kro/introduction.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Introduction"
+sidebar_position: 3
+---
+
+kro operates within a cluster using two primary components:
+
+1. The kro controller manager, which provides the core orchestration functionality
+2. ResourceGraphDefinitions (RGDs), which define templates for creating groups of related resources
+
+The kro controller manager watches for ResourceGraphDefinition custom resources and orchestrates the creation and management of the underlying Kubernetes resources defined in the template.
+
+kro simplifies complex resource management by allowing platform teams to define ResourceGraphDefinitions that encapsulate multiple related resources. Developers interact with simple custom APIs defined by the RGD schema, while kro handles the complexity of creating and managing the underlying resources.
+
+## ResourceGraphDefinition Workflow
```

Comment:
This seems a little out of place being before the install instructions. I'd say move it to the bottom of the page.